pkg/auth/dependency/webapp/service.go

```diff
@@ -231,6 +249,7 @@ func (s *Service) afterPost(state *State, graph *newinteraction.Graph, edges []n
 
 	// Case: finish
 	if finished {
+		s.Logger.Debugf("afterPost finished")
```

Comment:
Actually for these logs (and also the connection open/close logging), I think `TRACE` level is more suitable, otherwise the console would be spammed with logs that aren't useful most of the time. Let's review the appropriate log levels later.